modules/core/02-client/keeper/client_test.go

```diff
@@ -149,31 +149,34 @@ func (suite *KeeperTestSuite) TestUpdateClientTendermint() {
 			updateHeader = createPastUpdateFn(fillHeight, trustedHeight)
 		}, true, false},
 		{"valid duplicate update", func() {
-			clientID := path.EndpointA.ClientID
-
 			height1 := clienttypes.NewHeight(1, 1)
 
 			// store previous consensus state
 			prevConsState := &ibctm.ConsensusState{
 				Timestamp:          suite.past,
 				NextValidatorsHash: suite.chainB.Vals.Hash(),
 			}
-			suite.chainA.App.GetIBCKeeper().ClientKeeper.SetClientConsensusState(suite.chainA.GetContext(), clientID, height1, prevConsState)
+			path.EndpointA.SetConsensusState(prevConsState, height1)
 
 			height5 := clienttypes.NewHeight(1, 5)
 			// store next consensus state to check that trustedHeight does not need to be hightest consensus state before header height
```

Comment:
may as well fix this typo while we're at it? :D 


```suggestion
			// store next consensus state to check that trustedHeight does not need to be highest consensus state before header height
```